awx/main/tasks.py

```diff
@@ -279,6 +280,12 @@ def handle_setting_changes(setting_keys):
     logger.debug('cache delete_many(%r)', cache_keys)
     cache.delete_many(cache_keys)
 
+    if any([
+        setting.startswith('LOG_AGGREGATOR')
+        for setting in setting_keys
+    ]):
+        reconfigure_rsyslog()
```

Comment:
oh, so there's only 1 rsyslog running on each instance, so we don't get into the insanity of the full service restart that we used to have.

Reply:
Right, this is similar to what we do with memcached.  Each node runs rsyslog and talks to its own local service, and all of them are configured and kept up to date the same way via the settings broadcast update mechanism.